packages/apps/api/src/handlers/user.ts

```diff
@@ -371,44 +289,26 @@ export async function changePassword(
 
   const password: string = request.body.password;
   if (!isNonBlank(password)) {
-    response
-      .status(400)
-      .json(toMessage('All fields must be provided and non-blank'));
-    return;
+    throw new HttpError({
+      status: 400,
+      message: 'All fields must be provided and non-blank',
```

Comment:
Minor: should we return a more specific error message here? We know the password field is non-blank, are there any other fields we're expecting?

Reply:
cf the error message further down this diff

Reply:
I was just replicating existing behaviour here, but yeah I'll tweak this copy